tests/sycl/test_sycl_containers.sycl

```diff
@@ -294,6 +294,73 @@ TEST_F(sycl_containers_test, atomic_device_memory) {
     }
 }
 
+/// Test atomic access to local memory
+TEST_F(sycl_containers_test, atomic_local_ref) {
+
+    // Create the SYCL queue that we'll be using in the test.
+    cl::sycl::queue queue;
```

Comment:
Something like

```c++
if (queue.get_device().get_info<::sycl::info::device_type>() != ::sycl::info::device_type::gpu) {
    GTEST_SKIP();
}
```

is still missing. (For now let's only enable these things for GPUs. As I doubt that "local memory" is even defined for CPU OpenCL devices. May be worth checking though...)

Reply:
Yup, I also couldn't find any clear documentation on whether or not that exists.
I'll push the latest version here, which I think should pass all tests, but I'm still not understanding some issues I'm running into locally,